funnel/models/venue.py

```diff
@@ -14,7 +14,7 @@ class Venue(UuidMixin, BaseScopedNameMixin, CoordinatesMixin, db.Model):
 
     project_id = db.Column(None, db.ForeignKey('project.id'), nullable=False)
     project = db.relationship(Project,
-        backref=db.backref('venues', lazy='dynamic', cascade='all, delete-orphan', order_by='Venue.name'))
+        backref=db.backref('venues', lazy='dynamic', cascade='all, delete-orphan', order_by='Venue.seq'))
```

Comment:
Use [SQLAlchemy's ordering_list](https://docs.sqlalchemy.org/en/latest/orm/extensions/orderinglist.html) for the relationship.

Reply:
@jace ordering_list doesn't work properly with backref. it works if I remove backref from here and define `venues` in `Project` model.